plugins/UM3NetworkPrinting/resources/qml/MonitorPrintJobProgressBar.qml

```diff
@@ -34,15 +34,15 @@ Item
         {
             background: Rectangle
             {
-                color: "#f5f5f5" // TODO: Theme!
-                implicitHeight: visible ? 8 * screenScaleFactor : 0 // TODO: Theme!
+                color: UM.Theme.getColor("monitor_progress_bar_empty")
+                implicitHeight: visible ? 12 * screenScaleFactor : 0 // TODO: Theme!
```

Comment:
Is the change of this number intentional?

Reply:
Yup. Earlier designs of the monitor tab used a different progress bar style than the rest of Cura, this change brings it in line.